src/main/java/bio/terra/service/resourcemanagement/azure/AzureAuthService.java

```diff
@@ -142,13 +153,31 @@ public BlobServiceClient getBlobServiceClient(
   /** Obtain a secret key for the associated storage account */
   private String getStorageAccountKey(
       UUID subscriptionId, String resourceGroupName, String storageAccountResourceName) {
-    AzureResourceManager client = configuration.getClient(subscriptionId);
 
-    return client
-        .storageAccounts()
-        .getByResourceGroup(resourceGroupName, storageAccountResourceName)
-        .getKeys()
-        .get(0)
-        .value();
+    int timeoutSeconds = 86400;
```

Comment:
I was originally thinking we could use a time unit call to make this number a bit more readable, but it turns out the needed code is not really readable either 😢 
`Long.valueOf(TimeUnit.HOURS.toSeconds(24)).intValue();`

Reply:
`TimeUnit.HOURS.toSeconds(24)` already produces a `Long`!

Reply:
It gets mad about that for some reason!